src/Home/index.js

```diff
@@ -59,23 +81,85 @@ export const Home = () => (
 		<Header />
 		<Main>
 			<Content>
-				<Banner>
-					<BannerTitle isWide={false}>
-						Revolutionizing music with the blockchain
-					</BannerTitle>
-					Listen to Artist of the week<span>
-						<br />
-					</span>
-					<BannerText>
-						100% free streaming for listeners, industry best compensation for
-						musicians.
-					</BannerText>
-					<PrimaryButton to="https://musicoin.org/welcome" target="_blank">
-						Join the revolution
-					</PrimaryButton>
-					<ArrowButton to="/how-it-works">How it works</ArrowButton>
-				</Banner>
+				<Media marginTop={'125px'}>
+					<MediaImage src={mobileAppImage} order={0} width={477} height={450} />
+					<MediaContent>
+						<TertiaryHeading level={2} isPretty={true} color={'#FFFFFF'}>
+							Music For All
+						</TertiaryHeading>
+						<p>
+							95% of internet users can't listen to music online easily today
+							with expensive but closed streaming services. With the power of
+							blockchain, you can stream freely from more than 8 million
+							independent artists very soon. There's no ads. It is free,
+							shareable and artists get paid instantly.
+							{/*With the power of our blockchain platform, you can stream more
+							than 60,000 tracks, from more than 5000 independent artists.
+							There's no ads, it's free, and artists get paid instantly.*/}
+						</p>
+						<p>Try for yourself & discover new music now!</p>
+						{/*<Link to="/test" style={{backgroundColor:"transparent", backgroundSize:"50%", backgroundPosition: "center", backgroundRepeat:"no-repeat", backgroundImage:"url(./images/google-play-badge.png)"}}>test</Link>*/}
+						<PrimaryHeading
+							level={3}
+							isPretty={true}
+							color={'#f3921b'}
+							customSize={18}
+							customMargin="0px 0px 16px 0px"
+						>
+							<label>Musicoin Mobile App V1.0 (coming soon)</label>
```

Comment:
I suggest a link to the Beta Tester Program around "coming soon", although not so critical.